metrics-core/src/main/java/com/codahale/metrics/ConsoleReporter.java

```diff
@@ -317,4 +333,17 @@ private void printWithBanner(String s, char c) {
         }
         output.println();
     }
+
+    /**
+     * Print only if the attribute is enabled
+     * @param type Metric attribute
+     * @param status Status to be logged
+     */
+    private void printIfEnabled(MetricAttribute type, String status) {
+        if(isMetricAttributeDisabled(type)) {
```

Comment:
I think it would better with one condition statement. I also don't think the we need to use `isMetricAttributeDisabled` method as well.
```
if (!getDisabledMetricAttributes().contains(type)) {
    output.println(status);
}
```